docs/source/conf.py

```diff
@@ -33,7 +33,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.githubpages", "sphinx.ext.napoleon"]
+extensions = ["sphinx.ext.githubpages", "sphinx.ext.napoleon", "sphinx_click.ext"]
```

Comment:
sphinx_click needs to be installed with pypi. So you need to add it to the `setup` file within extra_requires: docs.

Reply:
Added now !